Dockerfile

```diff
@@ -8,21 +8,19 @@
 
 FROM aiidateam/aiida-core-with-services:2.5.0
 
+
 # To prevent the container to exit prematurely.
 ENV KILL_ALL_RPOCESSES_TIMEOUT=50
 
-
 USER root
-# Install statically linked CP2K which is a considerably newer release than Debian builtin.
-# The statically linked CP2K is a non-MPI binary, but we're running all tests with 1 MPI proc.
 RUN set -ex ; \
   apt-get update ; \
-  apt-get install -y --no-install-recommends openmpi-bin locate; \
-  wget --no-verbose -O /usr/bin/cp2k https://github.com/cp2k/cp2k/releases/download/v8.2.0/cp2k-8.2-Linux-x86_64.ssmp ; \
-  echo "1e6fccf901873ebe9c827f45fb29331f599772f6e6281e988d8956c7a3aa143c /usr/bin/cp2k" | sha256sum -c ; \
-  chmod +x /usr/bin/cp2k
+  apt-get install -y --no-install-recommends libsymspg1
 
 USER aiida
+
+RUN mamba create --yes -c conda-forge -n cp2k cp2k=9.1 && mamba clean --all -f -y
```

Comment:
Why don't you install the latest version of CP2K?

Reply:
The next step would be to test against multiple versions of cp2k,  to make sure the plugin is compatible with all of them.

I am sure the new ones will have issues, so to not blow up the PR I decided to make the change as small as possible.

Reply:
By the way, would it be possible to also put ARM builds there?

Reply:
> By the way, would it be possible to also put ARM builds there?

CP2K supports ARM with Linux and macOS. This is also tested on the Dashboard. However, those tests sill use the old Makefile build system. So, adding ARM to the Conda package should be doable, but might requires ironing out some CMake issues.

Reply:
> CP2K supports ARM with Linux and macOS. This is also tested on the Dashboard. However, those tests sill use the old Makefile build system. So, adding ARM to the Conda package should be doable, but might require ironing out some CMake issues.

It is not urgent, but if it could be done at some point - this would be great.